cfgov/unprocessed/js/modules/util/dom-events.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+/**
+ * Parses the event passed by the bind shortcut to create the event listener
+ * @param   {string}   evt      The type of event to watch for
+ * @param   {HTMLNode} elem     The element to watch for the passed event
+ * @param   {Function} callback The function to trigger on the event
+ */
+function _parseEvent( elem, evt, callback ) {
+  evt.split( /\s+/ ).forEach(
+    function( event ) {
+      elem.addEventListener( event, callback );
+    }
+  );
+}
+
+/**
+ * Shortcut for binding event listeners to elements
+ * @param  {HTMLNode} elem    The element to attach the event listener to
+ * @param  {Object}   options The options for the event listener
+ */
+function bindEvent( elem, options ) {
+  var callback;
+  for ( var evt in options ) {
+    if ( options.hasOwnProperty( evt ) ) {
+      callback = options[evt];
+      _parseEvent( elem, evt, callback );
+    }
+  }
+}
+
+/**
+ * Shortcut for triggering events on elements
+ * @param  {HTMLNode} target     The element to fire events on
+ * @param  {string}   type       The event type to create
+ * @param  {Object}   properties The options for the event
+ */
+function fireEvent( elem, type, properties ) {
+  var evt = document.createEvent( 'HTMLEvents' );
+
+  evt.initEvent( type, true, true );
```

Comment:
Should this be `initCustomEvent`? If this is for IE fallback, we'll probably want to wrap it some logic to check for support since `initCustomEvent` is deprecated https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/initCustomEvent